### PR TITLE
Update B01LER content

### PR DESCRIPTION
**Pull request by Jumace**
The content for the B01LER project was only a teaser, now we have proper links and a bit more information

### CODE REVIEW

File: src/_includes/content/boiler.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Introduction
+
+B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.
+
+B01LER provides pre-built components that exist in design in the form of a [component library in Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig)), as well as in [code in a repository](https://github.com/deven-org/B01LER-Kitchen), which is also [displayed in the web using Storybook](https://b01ler.onrender.com/). Both the components in design and in code are connected via a token system.
+
+A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
```
Comment (remmycat) [inline src/_includes/content/boiler.md:7]: That first sentence got quite long with lots of stuff happening, which makes it hard to read to me. I would reduce the number of links to the main two (not the internal one), to not distract too much with extra information here.

I'm also not certain if "in design" and "in code" work as a key phrase here.

How about this:

```suggestion
B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/B01LER-Kitchen).

The design and code implementations of the components are connected via a shared **token system**. A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
```

Rendered:

---

B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/B01LER-Kitchen).

The design and code implementations of the components are connected via a shared **token system**. A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.

File: src/_includes/content/boiler.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### Benefits of B01LER
+1. Increased efficiency in building design systems on brand, because semantic structure and architecture are pre-built
+2. Increased efficiency, through automated processes. Documentation, DEV Handover, and more
+3. Increased effectivity, because the time saved on building all the components can be used to build features right away
+4. Increased acceptance of DS because components work right from the beginning of project phase
+5. Immense cost savings, because the DS ramp-up phase is reduced from multiple months to days
+
+## Resources
+- [Browse through our website](https://boilerds.com/)
+- [Explore our Figma File](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig))
```
Comment (remmycat) [inline src/_includes/content/boiler.md:22]: The public link should probably be the main one:

```suggestion
- [Download our Figma File](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig) (Accenture-internal: [View on Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1))
```

**Review by remmycat — COMMENTED** [review]
I had two comments about the content, though they probably also partially apply to the B01ler Readme, I'm not sure if you want to edit anything here?

File: src/_includes/content/boiler.md (hunk 1 above)
Comment (Jumace) [reply src/_includes/content/boiler.md:7]: You are right, thanks for the suggestion, will go with that

File: src/_includes/content/boiler.md (hunk 2 above)
Comment (Jumace) [reply src/_includes/content/boiler.md:22]: Again a good point, will update the content

File: docs/assets/css/main.css [hunk 3]
```diff
@@ -28,6 +28,10 @@ body {
   }
 }
 
+strong {
+  font-weight: bold;
+}
+
```
Comment (remmycat) [inline docs/assets/css/main.css:34]: Like we talked about, maybe put this into richtext.scss? 🙂

**Review by remmycat — APPROVED** [review]

**Pull request merged**